README.md

```diff
@@ -374,6 +375,16 @@ Additionally, we would also advise that your retain the text saying "Powered by
 
 Everything else is made open according to the apache2 license. We look forward to seeing your data marketplace!
 
+## Advanced Features
+
+Ocean Market also includes a number of advance features that are suitable for an enterprise data market, such as:
+
+- Role based access control
+- Allow and deny lists
+- Free pricing
+
+[See our seperate guide on advance features](docs/advanceDocumentation.md)
```

Comment:
```suggestion
[See our seperate guide on advanced features](docs/advancedDocumentation.md)
```

so also change the file name to `docs/advancedDocumentation.md`. And maybe rather call it `docs/advanced-settings.md`?